modules/client/examples/05-encoders-decoders/01-grant-permission.ts

```diff
@@ -0,0 +1,69 @@
+/* MARKDOWN
+---
+title: Grant Permissions
+---
+
+## Grant a Permission
+
+Grants permission with the name (`permission`) to an address (`who`) to perform on a target contract (`where`).
+
+### Encoding
+*/
+
+import {
+  Client,
+  DaoAction,
+  IGrantPermissionParams,
+  IGrantPermissionDecodedParams,
+  Permissions
+} from "@aragon/sdk-client";
+import { context } from "../index";
+
+// Instantiates an Aragon OSx SDK client.
+const client: Client = new Client(context);
+
+const params: IGrantPermissionParams = {
+  who: "0x1234567890123456789012345678901234567890",
+  where: "0x1234567890123456789012345678901234567890",
+  permission: Permissions.UPGRADE_PERMISSION
+};
+
+const daoAddress: string = "0x1234567890123456789012345678901234567890";
+
+const action: DaoAction = await client.encoding.grantAction(daoAddress, params);
+console.log({ action });
+
+/* MARKDOWN
+Returns:
+
+```json
+{ grantPermission:
```

Comment:
If we change the name of the variable from `grantPermission`to action this is no longer correct and must be
```ts
{ action: 
...
}
```